src/loki/isochrone_action.cc

```diff
@@ -31,6 +31,16 @@ void check_distance(const google::protobuf::RepeatedPtrField<valhalla::Location>
     }
   }
 }
+
+const std::tuple<float, float>
+get_max_metrics(const google::protobuf::RepeatedPtrField<valhalla::Contour>& contours) {
+  float max_dist, max_time = 0.0f;
```

Comment:
@nilsnolde this is a bug, here max_dist undefined behavior. it will have random junk in it. when that random junk is a high value, this function will return that value

Reply:
sure, I thought so, tested it separately and it returned fine locally. well, that syntax then doesn't initialize both vars..